### DISCUSSION TITLE
Which will be the next LTS version?

### DISCUSSION THREAD

**Discussion by viliamjr** [laravel/framework#45039] [Q&A] [answered]
The Laravel 6 reached end-of-life (september 2022), which will be the next LTS version?

![image](https://user-images.githubusercontent.com/47677/203062026-72f1dc43-9437-49dd-92c2-b1dedc7c057b.png)

**Comment by Sophist-UK** [comment]
It might be worth posting a table of PHP support dates - because I think that will have a major influence. (PHP 8.0 goes out of support this week!!)

**Comment by henzeb** [comment, marked as answer]
As I understand it, they abandoned the LTS approach. Although laravel 9 will receive 2 years of support.

https://darkghosthunter.medium.com/laravel-goodbye-lts-356fd799e3c2

**Reply by viliamjr** [reply]
Thank you! Makes total sense.
"Prepare for a world without true LTS" 🤞🏼
